src/frontend/screens/Game/GamePage/index.tsx

```diff
@@ -232,9 +232,18 @@ export default observer(function GamePage(): JSX.Element | null {
               setGameInstallInfo(info)
             })
             .catch((error) => {
-              console.error(error)
-              window.api.logError(`${`${error}`}`)
-              setHasError({ error: true, message: `${error}` })
+              const errorMessage = t(
```

Comment:
Improved the error message here for better debugging next time the GamePage fails to load.